v2/apiserver/internal/core/mongodb/events_store_test.go

```diff
@@ -799,15 +844,15 @@ func TestEventsStoreDeleteMany(t *testing.T) {
 					return nil, errors.New("something went wrong")
 				},
 			},
-			assertions: func(events core.EventList, err error) {
+			assertions: func(err error) {
 				require.Error(t, err)
 				require.Contains(t, err.Error(), "error logically deleting events")
 				require.Contains(t, err.Error(), "something went wrong")
 			},
 		},
 
 		{
-			name: "error finding deleted events",
+			name: "error counting deleted events",
```

Comment:
This whole test case can go away if we don't make the extra db call to count affected events.